### ISSUE TITLE
Redundant constraints generated

### ISSUE THREAD

**Issue by etorreborre** [waddlaw/haskell-stack-trace-plugin#14]
@waddlaw I'm opening a new issue to deal with this redundant constraint warning that I observed after #11
```
 • Redundant constraints: (HasCallStack, HasCallStack)
    • In the type signature for:
           contractConfig :: (HasCallStack, HasCallStack, HasCallStack) =>
                             ContractConfig
    |
225 | contractConfig :: ContractConfig
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by abhin4v** [comment]
Possibly related is this unused imports warning:

```
warning: [-Wunused-imports]
    The qualified import of ‘GHC.Stack’ is redundant
      except perhaps to import instances from ‘GHC.Stack’
    To import instances alone, use: import GHC.Stack()
```
